application/src/test/java/org/opentripplanner/street/integration/WalkRoutingTest.java

```diff
@@ -45,18 +48,38 @@ class WalkRoutingTest {
   void shouldRouteAroundRoundabout() {
     var start = new GenericLocation(59.94646, 10.77511);
     var end = new GenericLocation(59.94641, 10.77522);
-    Assertions.assertDoesNotThrow(() -> route(roundabout, start, end));
+    Assertions.assertDoesNotThrow(() -> route(roundabout, start, end, dateTime, false));
+  }
+
+  @ParameterizedTest
+  @ValueSource(ints = { 0, 200, 400, 499, 500, 501, 600, 700, 800, 900, 999 })
+  void pathReversalWorks(int offset) {
+    var start = new GenericLocation(59.94646, 10.77511);
+    var end = new GenericLocation(59.94641, 10.77522);
+    var base = dateTime.truncatedTo(ChronoUnit.SECONDS);
+    var time = base.plusMillis(offset);
+    var results = route(roundabout, start, end, time, true);
+    Assertions.assertEquals(1, results.size());
```

Comment:
You should use static imports for the test methods. Also, nowadays we have the option of using `Truth.assertThat` for validating collection sizes.